AZ3166/src/cores/arduino/az_iot/provisioning_client/deps/RIoT/Reference/RIoT/Core/RiotCore.c

```diff
@@ -88,8 +88,46 @@ int RiotStart(uint8_t *CDI, uint16_t CDILen, const char *RegistrationId)
     uint32_t            length;
     uint32_t            dcType;
 
-    // Update subject common of alias certificate TBD data to input registration Id
-    x509AliasTBSData.SubjectCommon = RegistrationId;
+    // Verify registrationId input and given an empty input, generate one based on MAC address and firmware version of your DevKit
```

Comment:
File name change to DevKitRiotCore, function name too.

Reply:
I will refactor all DPS files in a separate PR.